tests/Application/.env

```diff
@@ -12,7 +12,7 @@ APP_SECRET=EDITME
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For a sqlite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
 # Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
-DATABASE_URL=mysql://root@127.0.0.1/sylius_%kernel.environment%?serverVersion=5.5
+DATABASE_URL=mysql://root:password@127.0.0.1/sylius_test?serverVersion=8.0
```

Comment:
```suggestion
DATABASE_URL=mysql://root@127.0.0.1/bitbag_sylius_blacklist_plugin_%kernel.environment%
```

Reply:
I think we should use a more self-descriptive naming strategy at this point 🤔 (e.g. during developing multiple plugins)